### PR TITLE
Added some specs

### PR DESCRIPTION
**Pull request by jamesbowles**

### CODE REVIEW

**Comment by nigelramsay** [comment]
Do you want to drop in a `.travis.yml` file to run these new tests?

File: easy_reference_data.gemspec [hunk 1]
```diff
@@ -18,4 +18,9 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_runtime_dependency 'rails', '>= 3.0.0'
+
+  gem.add_development_dependency 'rspec', '~> 3.6.0'
+  gem.add_development_dependency 'sqlite3', '~> 1.3'
+  gem.add_development_dependency 'database_cleaner', '~> 1.6'
+  gem.add_development_dependency 'byebug'
```
Comment (nigelramsay) [inline easy_reference_data.gemspec]: can `byebug` now be removed?

File: spec/support/schema.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'active_record'
+require 'benchmark'
```
Comment (nigelramsay) [inline spec/support/schema.rb]: Do you really need `benchmark`?

File: spec/support/schema.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+ActiveRecord::Schema.define do
+  self.verbose = false
+
+  create_table :users, force: true do |t|
+    t.integer :system_code
+    t.string :name
+    t.string :email
+    t.timestamps
```
Comment (nigelramsay) [inline spec/support/schema.rb]: Are `timestamps` necessary?

**Comment by jamesbowles** [comment]
@nigelramsay Added a .travis.yml file, the build is here. https://travis-ci.org/jamesbowles/easy_reference_data/
I have just enabled build for this repo in travis: https://travis-ci.org/AbleTech/easy_reference_data

**Pull request merged**